Scenarios/PrivateCloud/AVS-PrivateCloud/ARM/PrivateCloud.deploy.json

```diff
@@ -52,6 +59,20 @@
           "clusterSize": "[parameters('ManagementClusterSize')]"
         }
       }
+    },
+    {
+      "condition": "[not(parameters('TelemetryOptOut'))]",
+      "type": "Microsoft.Resources/deployments",
```

Comment:
Can't see the Bicep change for this in the changeset.

Reply:
@prasad3017 had done this one earlier in the first test PR, had not done ARM in that one

Reply:
Ok, in future try to keep Bicep & ARM changesets together, otherwise it makes it super hard to track down where changes resulted from.